src/lib.rs

```diff
@@ -41,13 +42,15 @@ extern crate tempfile;
 extern crate zdaemon;
 extern crate zfilexfer;
 extern crate hostname;
+extern crate interfaces;
+extern crate ipnetwork;
 
 #[macro_use]
 mod ffi_helpers;
 mod command;
 mod project;
 mod directory;
-mod error;
+pub mod error;
```

Comment:
Oh  damn. Stupid me! I can patch that and you can rebase, or you could just stick another commit onto this PR?